orttraining/orttraining/python/training/optim/_ds_modifier.py

```diff
@@ -23,6 +25,142 @@
 multi_tensor_applier = MultiTensorApply(2048 * 32)
 
 
+def _get_sources(function) -> str:
+    return inspect.getsource(function)
+
+
+def _compare_str_list(src, dest):
+    return "".join(src) == "".join(dest)
+
+
+def _get_normalized_str_list(source_str):
+    lines = source_str.split("\n")
+    return lines
+
+
+_DS_VERSION_TO_SOURCES_MAP: Dict[str, Dict[str, List[str]]] = {
+    "0.9.2": {
+        "has_overflow_serial": [
```

Comment:
Can this code be in a txt file? Or it could also be in a .py file as a function and we can compare the function bodies of deepspeed and this function.

Reply:
Ok only file is needed. Updated.